doc/source/configuration/external_satpy.rst

```diff
@@ -23,32 +23,40 @@ It is your responsibility to make sure the setting points to a suitable Satpy
 package: If the given path doesn't point to a Python package directory or not to
 one providing Satpy, the application may exit immediately throwing Exceptions.
 
-Using Extra Readers
-===================
+Using Extra Satpy component configuration
+=========================================
+
+SIFT can use external `Satpy component configuration <https://satpy.readthedocs.io/en/stable/config.html#component-configuration>`_ folder,
```

Comment:
this change does not work in my local repositroy. When I `make html` I got `D:\git\sift_fork\doc\source\configuration\external_satpy.rst:29: WARNING: undefined label: 'satpy:component-configuration'` and I was not able to resolve it.

Reply:
Before giving up on it, could you try `component_configuration` (with an underscore) and see if that works better? I've had mixed success using references like this between projects with intersphinx. I think in one case I ended up just doing the URL as you have it here because intersphinx would not find the reference no matter what I did.

Reply:
That was it, the underscore. Thanks.